projects/bp-gallery/src/views/gallery/shared/helpers/scanner-hook.ts

```diff
@@ -14,6 +15,8 @@ const useScanner = () => {
   const [scanners, setScanners] = useState<string[]>();
   const [selectedScannerId, setSelectedScannerId] = useState<number>(0);
 
+  const [shouldCrop, setShouldCrop] = useState<boolean>(false);
```

Comment:
Maybe useful but I'm not sure how important: Persisting the shouldCrop state e.g. via the localStorage

Reply:
Hmm, if we want to enforce that he spends a bit more time on each picture, i think it is quite acceptable that he needs to manually configure it every time he scans 😅